static/monorail-2.0.yaml

```diff
@@ -60,6 +60,23 @@ definitions:
         type: string
       username:
         type: string
+  hooks.2.0_HooksBase:
```

Comment:
This schema is still needed for the swagger UI and documentation.  It is not needed for validation although the validator will still validate against it.